LeanAide/PaperCodes.lean

```diff
@@ -10,13 +10,24 @@ Each function corresponds to a specific JSON schema element and generates the ap
 -/
 open Lean Meta Qq Elab Term
 
+-- #eval polyTraceIO.on ()
```

Comment:
I've added some commented lines that demonstrate the usage, let me know if this kind of usage is what you were looking for?